readthedocs/settings/dev/celery/builds.py

```diff
@@ -0,0 +1,27 @@
+"""Local setting for celery build workers (without DB access)"""
+
+import os
+
+from ..base import CommunityDevSettings
+
+
+# Load ``local_settings`` first to avoid ``DATABASES`` being override
+if not os.environ.get('DJANGO_SETTINGS_SKIP_LOCAL', False):
+    try:
+        # pylint: disable=unused-wildcard-import
+        from ..local_settings import *  # noqa
+    except ImportError:
+        pass
```

Comment:
This isn't quite what we want. This will load all the settings from the `local_settings` module, and then the base settings will just override all of the local settings. This block should still happen after `load_settings` is called, but should instead default `DATABASES` to the empty dict that you're using here. This could be manage using a couple of classes and calls to `load_settings`.

Reply:
I've added a pattern for altering the setting module after local_settings.